app/src/app/session/email-verification.dialog.html

```diff
@@ -0,0 +1,11 @@
+  <div [ngSwitch]="emailSent">
```

Comment:
Indentation of this line is wrong. It must be aligned with its closing tag line 11.